Gems/ScriptCanvas/Code/Builder/ScriptCanvasBuilderDataSystemBus.h

```diff
@@ -10,48 +10,86 @@
 
 #include <AzCore/EBus/EBus.h>
 #include <Builder/ScriptCanvasBuilder.h>
+#include <ScriptCanvas/Asset/RuntimeAsset.h>
 
 namespace ScriptCanvasBuilder
 {
-    enum class BuilderDataStatus
+    // sources...
+    enum class BuilderSourceStatus
     {
         Failed,
         Good,
         Removed,
         Unloadable,
     };
 
-    struct BuildResult
+    struct BuilderSourceResult
```

Comment:
These are the buses used by the EditorScriptCanvas component and the classes in the Interpreter widget to communicate with the SC Builder Data System.